technology-considerations/s3-user-access-to-objects.md

```diff
@@ -0,0 +1,121 @@
+# Restricting User Access To S3 Objects
+
+## Problem
+
+Ensure that TDR users are not able to overwrite/view another user’s objects in the TDR AWS S3 upload bucket.
+
+Potential options to resolve the problem are to use:
+* AWS Cognito;
+* Signed URLS.
+
+The purpose of the spike was to see whether using AWS Cognito was a feasible option.
+
+AWS Cognito has some advantages over using signed URLs:
```

Comment:
Another big advantage of Cognito is that if we used signed URLs, we would have to generate one URL per file. That means generating thousands of URLs for large consignments.

Reply:
Added